CESC-submission/paper.tex

```diff
@@ -208,64 +208,61 @@ \section{Impact of Financial Incentives on Service Delivery}
 re-encryption service. \\
 
 
-\noindent{\bf Monetary incentives in the NuCypher network.} Ursulas are rewarded for the re-encryption service by using two sources; the first is the service fees collected from the policy owner Alice, while the second is inflation rewards, i.e., newly minted NU tokens, coming from the NuCypher network. The latter will be provided during the 
-early stage of the network operation to encourage adoption. It is expected that these rewards will disappear as the token cap is reached, similar to other cryptocurrencies in the space.
-
+\noindent{\bf Monetary incentives in the NuCypher network.} Ursulas are rewarded for their re-encryption service by using two sources; the first is the service fees collected from the policy owner Alice, while the second is a subsidy in the form of newly minted NU tokens distributed on a schedule enforced by the NuCypher protocol. The latter will be provided during the early stage of the network operation to encourage service-provider participation and sustain their operations in lieu of a mature fee market.
```

Comment:
Ursulas are rewarded for the re-encryption service ->  Ursulas are rewarded for their re-encryption service
distribted -> distributed (change from arj)